File: src/desktop/src/com/lavaeater/desktop/DesktopLauncher.java
package com.lavaeater.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import com.lavaeater.TurboRakettiUltra;

public class DesktopLauncher {
    public static void main (String[] arg) {
        LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
        config.height = 1080;
        config.width = 1920;
        //config.fullscreen = true;
        new LwjglApplication(new TurboRakettiUltra(), config);
    }
}

File: kftw/src/desktop/src/com/lavaeater/kftw/desktop/DesktopLauncher.kt
package com.lavaeater.kftw.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import com.lavaeater.kftw.KidsFromTheWasteLandGame

object DesktopLauncher {
    @JvmStatic
    fun main(arg: Array<String>) {
        val config = LwjglApplicationConfiguration()
        config.fullscreen = false
        config.width = 2048
        config.height = 1536
        config.vSyncEnabled = true
        LwjglApplication(KidsFromTheWasteLandGame(), config)
    }
}
